src/utils/router.js

```diff
@@ -40,4 +54,18 @@ const router = new Router({
   ]
 })
 
+router.beforeEach((to, from, next) => {
+  if (to.matched.some(route => route.meta.requiresAuth)) {
+    // this route requires auth, check if logged in
+    // if not, redirect to login page.
+    if (!store.getters['users/get_is_authenticated']) {
```

Comment:
I have removed is_authenticated from the store, since we haven't used it before.
You can either check:
- if the user_id is null
- or add is_authenticated back to the store